packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerHelpers.ts

```diff
@@ -6,14 +6,15 @@ export interface LayerInputParams {
   layerPermissions?: Permission[];
   authorizedAccountIds?: string;
   authorizedOrgId?: string;
+  hash?: string;
```

Comment:
Does this belong in this interface? Seems like the other fields in this interface are parameters to the layer CFN template, but this is for our own bookkeeping